index.js

```diff
@@ -635,9 +641,9 @@ process.on('SIGTERM', stop);
 function initIface(name, protocol) {
     let url;
     if (protocol === 'binrpc') {
-        url = 'xmlrpc_bin://' + (config.initAddress || config.listenAddress) + ':' + config.binrpcListenPort;
```

Comment:
why remove this? listenAdress is needed if hm2mqtt runs in a vm with nat networking or in a container that exposes the listenPort to the hosts interface

Reply:
It's actually the other way around. In a Docker environment you can and will have multiple network interfaces. It's therefore a good idea to bind the socket to `0.0.0.0` a.k.a. "listen on all interfaces". This happens [here](https://github.com/hobbyquaker/hm2mqtt.js/blob/be5b3a24bb90dc772a626e30f87bf170142c2f11/index.js#L1050) in your script.

The line from this comment creates the url-string that is sent to the CCU. If you don't specify the `--init-address` it will default to the information form your `firstip.js` script, otherwise (for e.g. NAT-ed Docker) the user has to provide the real external IP and take care of proper port forwarding.

You can leave the line in here but combined with the change from line 44, this makes no semantical sense anymore.